Gemfile

```diff
@@ -193,9 +193,6 @@ end
 group :scheduler, :manageiq_default do
   gem "rufus-scheduler"
 end
-# rufus has et-orbi dependency
-# this is temporary inclusion till we can get ConvertTimeToEoTime patch into upstream:
-gem "et-orbi"
```

Comment:
We should probably require at least v1.2.2, right?

Reply:
Even if we only needed to include this because we had to patch it? So now the patch is gone, this shouldn't be necessary either?

Reply:
With the patch gone, someone could get any version < v1.2.2 and it would be broken, right?

Reply:
ugh okay fine

Reply:
i mean they'd figure it out eventually

Reply:
We probably want ">= 1.2.2" to avoid being locked to v1.2. We really just care about the minimum, right?